src/Serilog.Sinks.NLog/LoggerConfigurationsNLogExtensions.cs

```diff
@@ -35,14 +38,17 @@ public static class LoggerConfigurationsNLogExtensions
         public static LoggerConfiguration NLog(
             this LoggerSinkConfiguration loggerConfiguration,
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
-            IFormatProvider formatProvider = null)
+            IFormatProvider formatProvider = null,
+            string outputTemplate = DefaultOutputTemplate)
```

Comment:
The `formatProvider`/`outputTemplate` args are normally in the reverse order (template first); perhaps we switch the order here and just bump the major version of the package to reflect that there's a breaking change?